ui/src/components/SampleView/SampleImage.jsx

```diff
@@ -500,7 +500,7 @@ export default class SampleImage extends React.Component {
       this.canvas.discardActiveObject();
     }
 
-    showContextMenu(true, ctxMenuObj, e.offsetX, e.offsetY);
```

Comment:
`offsetX/Y` is relative to the container (here the sample view container). By rendering the menu under `document.body`, and using `pageX/Y` instead, we make the positioning/stacking/... more robust. For instance, we could now put `overflow: hidden` on the sample view container if we ever need to.